src/module.c

```diff
@@ -1019,9 +1019,14 @@ RedisModuleCommand *moduleCreateCommandProxy(struct RedisModule *module, sds dec
 
 /* Register a new command in the Redis server, that will be handled by
  * calling the function pointer 'cmdfunc' using the RedisModule calling
- * convention. The function returns REDISMODULE_ERR if the specified command
- * name is already busy or a set of invalid flags were passed, otherwise
- * REDISMODULE_OK is returned and the new command is registered.
+ * convention.
+ *
+ * The function returns REDISMODULE_ERR in these cases:
+ * - The specified command is already busy.
+ * - The command name contains some chars we want to block.
```

Comment:
```suggestion
 * - The command name contains some chars that are not allowed.
```

IIRC we need to have a blank line before a bulleted list in order for the markdown to render it correctly, please look into it.

Reply:
i see RM_ReplyWithCallReply is doing fine (RedisModule_ReplyWithCallReply: https://redis.io/docs/reference/modules/modules-api-ref/):
```
 * Return:
 * - REDISMODULE_OK on success.
 * - REDISMODULE_ERR if the given reply is in RESP3 format but the client expects RESP2.
 *   In case of an error, it's the module writer responsibility to translate the reply...
int RM_ReplyWithCallReply(RedisModuleCtx *ctx, RedisModuleCallReply *reply) {
```

Reply:
i guess i'm wong.
@zuiderkwast am i confusing it with something?

Reply:
Perhaps it's different now with the new redis.io website. I think it's using a different markdown renderer.

Reply:
@itamarhaber ^^?

(for me it's a win-win, since i think it's more readable in the source without the blank line, and evidently that's more natural to people, so it was often overlooked)

Reply:
(For me it's a win (not win-win) since it was easily overlooked. I like the blanks though since structurally, a list is never be embedded in a paragraph and blank lines are delimiting paragraphs. However the difference between win and win-win is not important. :grin:)